src/main/java/org/kiwiproject/dropwizard/util/health/UnknownPropertiesHealthCheck.java

```diff
@@ -28,8 +29,13 @@ protected Result check() throws Exception {
         }
 
         return newUnhealthyResultBuilder(HealthStatus.INFO)
-                .withMessage("%d unknown properties detected", handler.getUnknownPropertyCount())
-                .withDetail("unexpectedPaths", handler.getUnexpectedPaths())
+                .withMessage(unknownPropertiesMessage(handler.getUnknownPropertyCount()))
+                .withDetail("unexpectedPaths", handler.getUnexpectedPropertyPaths())
                 .build();
     }
+
+    private String unknownPropertiesMessage(long count) {
+        checkPositive(count, "expected 1 or higher");
+        return count == 1 ? "1 unknown property detected" : count + " unknown properties detected";
```

Comment:
```suggestion
        return count == 1 ? "1 unknown property detected" : (count + " unknown properties detected");
```

When reading this, adding the `( )` makes it 100% clear